joins.sql

```diff
@@ -0,0 +1,84 @@
+SELECT *
+FROM users;
+
+SELECT *
+FROM posts
+WHERE user_id = 100;
+
+SELECT posts.*, users.first_name, users.last_name
+FROM posts
+  INNER JOIN users ON posts.user_id = users.id
+WHERE posts.user_id = 200;
+
+SELECT posts.*, users.username, users.last_name
+FROM posts
+  INNER JOIN users ON posts.user_id = users.id
+WHERE users.first_name = 'Norene' and users.last_name = 'Schmitt';
+
+SELECT users.username
+FROM users
+  INNER JOIN posts ON users.id = posts.user_id
+WHERE posts.created_at > '2015-01-01';
+
+SELECT posts.title, posts.content, users.username
+FROM posts
+  INNER JOIN users ON posts.user_id = users.id
+WHERE users.created_at < '2015-01-01';
+
+SELECT posts.title AS "Post Title", comments.*
+FROM comments
+  INNER JOIN posts ON comments.post_id = posts.id;
+
+SELECT posts.title AS post_title,
+  posts.url AS post_url,
+  comments.body AS comment_body,
+  comments.*
+FROM comments
+  INNER JOIN posts ON comments.post_id = posts.id
+WHERE comments.created_at < '2015-01-01';
+
+SELECT posts.title AS post_title,
+  posts.url AS post_url,
+  comments.body AS comment_body,
+  comments.*
+FROM comments
+  INNER JOIN posts ON comments.post_id = posts.id
+WHERE comments.created_at > '2015-01-01';
+
+SELECT posts.title AS post_title,
+  posts.url AS post_url,
+  comments.body AS comment_body,
+  comments.*
+FROM comments
+  INNER JOIN posts ON comments.post_id = posts.id
+WHERE comments.body LIKE '%USB%';
+
+SELECT posts.title AS post_title,
+  users.first_name,
+  users.last_name,
+  comments.body AS comment_body
+FROM comments
+  INNER JOIN users ON comments.user_id = users.id
```

Comment:
users should be joined on posts, currenlty the user's first and last name are the commenter's